README.md

```diff
@@ -39,7 +39,40 @@ It certainly doesn't help that the first option explained in `man tar` is:
 There seems to be room for simpler help pages, focused on practical examples.
 How about:
 
-![Animated SVG of the tldr client displaying the tar command.](images/tldr.svg)
+![Screenshot of the tldr client displaying the tar command.](images/tldr.png)
+
+```shell-session
```

Comment:
```suggestion
```shell
```
Just a small suggestion, this wouldn't do much but would make some parts colourful. Also, From what I remember `shell-session` isn't a valid fenced code block language.

Reply:
That's the downside of markdown, there's no standardization. Gitlab supports shell-session, I inappropriately assumed that github did as well. 

I'd like to note different behaviors. 
shell:
shell only colors the "*.html" in the last example.

console:
console colors all output from the command. This seems like the more appropriate selection.

Reply:
Can we do the same for the above `man tar` example?

Reply:
https://github.com/tldr-pages/tldr/pull/8720#issuecomment-1268686583 Should we just throw out the man tar example?